standard_reports/expiring-items/latest/report-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "is_custom": false,
-  "version": "2.6.1",
+  "version": "2.6.2",
```

Comment:
Why bump version here ? Nothing changed and we already bumped version when adding BoaJS type here

Reply:
I was just trying to retrace my steps on why I bumped versions, and I think it was [this file specifically](https://github.com/msupply-foundation/open-msupply/pull/7059/files#r2036412763) which didn't have the version bumped when changed to BoaJS, and also noticed that the reports.json file wasn't regenerated (so all my reports were staying as extism when I reinitialised my DB), I assumed they all needed a version bump.

But I didn't double check - it was only expiring items which was already on 2.6.1 and needed a bump, the rest is redundant.